pwiz_tools/Skyline/Model/ImportPeakBoundaries.cs

```diff
@@ -100,26 +101,57 @@ public enum Field { modified_peptide, filename, apex_time, start_time, end_time,
 
         // ReSharper disable LocalizableElement
 
+        private static string[] _peptide_synonyms; // Cache for performance reasons (L10N stuff is expensive)
         public static string[] PEPTIDE_SYNONYMS
         {
             get
             {
-                return new[] 
+                if (_peptide_synonyms == null)
                 {
-                    "PeptideModifiedSequence", "ModifiedSequence", "FullPeptideName", "EG.ModifiedSequence",
-                    ColumnCaptions.PeptideModifiedSequence, ColumnCaptions.ModifiedSequence
-                };
+                    var headers = new List<string>()
+                    {
+                        "PeptideModifiedSequence", "ModifiedSequence", "FullPeptideName", "EG.ModifiedSequence",

```

Comment:
Are the first 2 entries duplicated by the code below? The old code would have duplicated them also, but only when running with English settings. Otherwise, it allowed us to support both English and the current translation, but not any translated header from any language. With the new code, I think you only want the 3rd and 4th values in this initializer.

Reply:
I don't believe so, no. "PeptideModifiedSequence" vs "Peptide Modified Sequence".

Reply:
Presumably from others, yes, and also probably a good guess. I do a similar thing in transition list reading - accept versions of our own known headers with spaces stripped out.  

Also I just realized that there are other columns that need the all languages treatment.